lisa/features/gpu.py

```diff
@@ -207,6 +212,9 @@ def _install_cuda_driver(self, version: str) -> None:
             self._node.os.install_packages(install_packages, signed=False)
 
         elif isinstance(self._node.os, Ubuntu):
+            cuda_package_name = "cuda-drivers"
+            cuda_drivers_package_pattern = re.compile(r"cuda-drivers-(\d+)")
```

Comment:
better to match the whole string, for more robust logic

Reply:
Updated, testing.

Reply:
tested